tasks/deploy.py

```diff
@@ -59,6 +62,16 @@ def deploy(
     aws_account = cfg.get_aws().get_account()
     flags.setdefault("ddinfra:env", "aws/" + aws_account)
 
+    # Verify image deployed and not outdated in s3
+    if deploy_job is not None and pipeline_id is not None:
+        cmd = f'inv -e check-s3-image-exists --pipeline-id={pipeline_id} --deploy-job={deploy_job}'
+        cmd = tool.get_aws_wrapper(aws_account) + cmd
+        output = ctx.run(cmd)
```

Comment:
If you use `ctx.run(cmd)` without any other parameters, if the executed cmd fails (non-zero exit code) it will automatically fails the whole python execution.

If you want output to include the result and the exit_code of the command even when it fail you should use `ctx.run(cmd, warn=True)`.